src/components/sheet/SlackSheet.js

```diff
@@ -156,7 +159,25 @@ export default forwardRef(function SlackSheet(
     yPosition.value = event.contentOffset.y;
   });
 
-  const bg = backgroundColor || colors.white;
+  const { backgroundColors, colorMode } = useContext(ColorModeContext);
+  const accentColorContextValue = useContext(AccentColorContext);
+
+  let bg = backgroundColor;
```

Comment:
I think this would read better 
`
bg = colors.white
if (backgroundColor){
bg = backgroundColor
}
`

Reply:
ig this would need to check that its not 'accent' as well